<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>970</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>970</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>投资组合优化</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>已添加股票</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>60</y>
     <width>589</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>添加股票组合</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="text">
     <string>添加4只股票可启动程序，最多添加8只。（相关数据截止于2023年4月1日）</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>88</y>
     <width>641</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="bondBtn">
      <property name="text">
       <string>债券</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="stockBtn">
      <property name="text">
       <string>股票</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="masBtn">
      <property name="text">
       <string>中小板</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QProgressBar" name="progressBar">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>投资品类</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>品类细分</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="createBtn">
      <property name="text">
       <string>创业板</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="mainBtn">
      <property name="text">
       <string>主板</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="futureBtn">
      <property name="text">
       <string>期货</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="scBtn">
      <property name="text">
       <string>科创板</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>301</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBtn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>380</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startCalcBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>380</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始计算</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="calcProgressBar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>639</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
